pkg/internal/webhook/validator_test.go

```diff
@@ -56,14 +57,16 @@ func Test_validate(t *testing.T) {
 	})
 	tests := map[string]struct {
 		crp               runtime.Object
-		expectedWarnings  admission.Warnings
-		wantsErr          bool
 		webhooks          []approver.Webhook
 		registeredPlugins []string
+
+		expectedWarnings admission.Warnings
+		expectedError    *string
 	}{
 		"if the object being validated is not a CertificateRequestPolicy return an error": {
-			crp:      &corev1.Pod{},
-			wantsErr: true,
+			crp: &corev1.Pod{},
+
+			expectedError: pointer.String("expected a CertificateRequestPolicy, but got a *v1.Pod"),
```

Comment:
I added the literal errors in these tests, this way we know that these values do not change in this PR.

Reply:
I am on the edge with comparing error strings, but here the errors can come from different sources, so makes sense 👍🏼